ygot/struct_validation_map.go

```diff
@@ -414,14 +414,28 @@ func copyPtrField(dstField, srcField reflect.Value) error {
 	// Check for struct ptr, or ptr to avoid panic.
 	if util.IsValueStructPtr(srcField) {
 		var d reflect.Value
-		d = reflect.New(srcField.Type().Elem())
+
+		// If the destination value is non-nil, then we maintain its contents
```

Comment:
Suggest adding something about maintaining existing values & overwrite behavior in the function comment.

Reply:
Done.